docs/pages/getting-started/4-connecting-crm.rst

```diff
@@ -0,0 +1,216 @@
+
+.. code-block:: text
+
+    did:nuts:2mF6KT6eiSx5y2fwTP4Y42yMUh91zGVkbu4KMARvCJz9
+
+The DID we're about to create is your *vendor DID*. It will be used in the all of the next steps.
+For the API calls that will need to be made to the Nuts node, we'll use ``<internal-node-address>`` as the address where the internal API's are exposed.
+Consult the :ref:`configuration reference <nuts-node-config>` on how to configure the node address.
+
+.. code-block:: text
+
+    POST <internal-node-address>/internal/vdr/v1/did
+    {
+        "selfControl": true,
+        "assertionMethod": true,
+        "capabilityInvocation": true
+    }
+
+The request above instruct the node to create a new DID and DID Document. The DID Document will be published to all other nodes.
+The node will generate a new keypair and store it in the crypto backend.
+The options above will instruct the node to allow the DID Document to be changed by itself (``selfControl = true AND capabilityInvocation = true``) and that the DID can be used to issue credentials (``assertionMethod = true``).
+If all is well, the node will respond with a DID Document similar to:
+
+.. code-block:: json
+
+    {
+      "@context": [ "https://www.w3.org/ns/did/v1" ],
+      "id": "did:nuts:2mF6KT6eiSx5y2fwTP4Y42yMUh91zGVkbu4KMARvCJz9",
+      "verificationMethod": [
+        {
+          "id": "did:nuts:2mF6KT6eiSx5y2fwTP4Y42yMUh91zGVkbu4KMARvCJz9#_TKzHv2jFIyvdTGF1Dsgwngfdg3SH6TpDv0Ta1aOEkw",
+          "controller": "did:nuts:2mF6KT6eiSx5y2fwTP4Y42yMUh91zGVkbu4KMARvCJz9",
+          "type": "JsonWebKey2020",
+          "publicKeyJwk": {
+            "crv": "P-256",
+            "x": "38M1FDts7Oea7urmseiugGW7tWc3mLpJh6rKe7xINZ8",
+            "y": "nDQW6XZ7b_u2Sy9slofYLlG03sOEoug3I0aAPQ0exs4",
+            "kty": "EC"
+          }
+        }],
+      "capabilityInvocation": [
+        "did:nuts:2mF6KT6eiSx5y2fwTP4Y42yMUh91zGVkbu4KMARvCJz9#_TKzHv2jFIyvdTGF1Dsgwngfdg3SH6TpDv0Ta1aOEkw"
+      ],
+      "assertion": [
+        "did:nuts:2mF6KT6eiSx5y2fwTP4Y42yMUh91zGVkbu4KMARvCJz9#_TKzHv2jFIyvdTGF1Dsgwngfdg3SH6TpDv0Ta1aOEkw"
+      ],
+      "service": []
+    }
+
+The ``id`` at the top level needs to be extracted and stored as your vendor DID.
+In the example above this would be ``did:nuts:2mF6KT6eiSx5y2fwTP4Y42yMUh91zGVkbu4KMARvCJz9``
+
+Setting vendor contact information
+==================================
+
+Things can go wrong: a node is misbehaving or a DID Document is conflicted.
+If the node operator is not resolving the problem it's extremely convenient if others can contact the node operator and relay the problem.
+For this use-case, Nuts supports the registration of node contact information. The contact information will be added to a DID Document as a service.
+A convenience API is available to add the contact information to a DID Document. The vendor DID should be used for this.
+
+.. code-block:: text
+
+    PUT <internal-node-address>/internal/didman/v1/did/{did}/contactinfo
```

Comment:
maybe we can use 1 style of parameterization of the URL ( <...> vs {...} )

Reply:
using <> now